uat/custom-components/client-java-sdk/src/main/java/com/aws/greengrass/testing/mqtt5/client/GRPCClient.java

```diff
@@ -42,6 +42,9 @@ class MqttReceivedMessage {
         /** Optional payload format indicator. */
         private Boolean payloadFormatIndicator;
 
+        /** Optional message expiry interval. */
+        private Long messageExpiryInterval;
```

Comment:
Here is use Long in other place Interger, why ?

Reply:
in our proto file we are using Integer, but sdk is using Long, so I use the Long variable in ReceivedMessage as the final instance.